pyEDAA/UCIS/Cobertura.py

```diff
@@ -94,40 +103,47 @@ class Package:
 
 	name: str
 	classes: Dict[str, Class]
-	lines_valid: int
-	lines_covered: int
+	linesValid: int
+	linesCovered: int
 
 	def __init__(self, name: str):
 		self.name = name
 		self.classes = {}
-		self.lines_valid = 0
-		self.lines_covered = 0
+		self.linesValid = 0
+		self.linesCovered = 0
 
-	def add_statement(self, class_name: str, source_file: str, line: int, count: int) -> None:
-		try:
-			self.classes[class_name].add_statement(line, count)
-		except KeyError:
-			self.classes[class_name] = Class(source_file)
-			self.classes.get(class_name).add_statement(line, count)
+	def addClass(self, coberturaClass: Class):
+		assert coberturaClass.name not in self.classes, "Duplicated class name"
+		self.classes[coberturaClass.name] = coberturaClass
 
-		self.lines_valid += 1
+	def refreshStatistics(self) -> None:
+		self.linesValid = 0
+		self.linesCovered = 0
 
-		if count:
-			self.lines_covered += 1
+		for coberturaClass in self.classes.values():
+			self.linesCovered += coberturaClass.linesCovered
+			self.linesValid += coberturaClass.linesValid
 
-	def get_xml_node(self) -> etree.Element:
-		classes_node = etree.Element("classes")
-		package_node = etree.Element("package")
-		package_node.append(classes_node)
-		package_node.attrib["name"] = self.name
-		package_node.attrib["complexity"] = "0"
-		package_node.attrib["branch-rate"] = "0"
-		package_node.attrib["line-rate"] = str(self.lines_covered / self.lines_valid)
+	def getXmlNode(self) -> etree.Element:
+		classesNode = etree.Element("classes")
+		packageNode = etree.Element("package")
+		packageNode.attrib["name"] = self.name
+		packageNode.attrib["complexity"] = "0"
+		packageNode.attrib["branch-rate"] = "0"
 
-		for (class_name, class_data) in self.classes.items():
-			classes_node.append(class_data.get_xml_node())
+		try:
```

Comment:
Please change like the division above.

Reply:
Fixed in https://github.com/edaa-org/pyEDAA.UCIS/pull/7/commits/a3989380939cc40dad5c58f093cfe2d9e3aa499f

Reply:
Checked.